lua/treesitter-context.lua

```diff
@@ -65,6 +65,9 @@ local ns = api.nvim_create_namespace('nvim-treesitter-context')
 --- @type TSNode[]?
 local previous_nodes
 
+--- @type Context[]
```

Comment:
These should be per buffer.

```suggestion
--- @type table<integer,Context[]>
```